ctapipe/io/eventsource.py

```diff
@@ -198,3 +196,37 @@ def __enter__(self):
 
     def __exit__(self, exc_type, exc_val, exc_tb):
         pass
+
+    @classmethod
+    def from_url(cls, url, *args, **kwargs):
```

Comment:
Maybe also call it `for_`? Same reason as above.

Reply:
Actually, I wouldn't call it url if it does not support urls.

Reply:
It is referring to the `input_url` though